### PR TITLE
feat: import IAB minutes from the IAB wordpress site

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: ietf/meeting/migrations/0005_iabminutes.py [hunk 1]
```diff
@@ -0,0 +1,959 @@
...
+    DocAlias = apps.get_model("doc", "DocAlias")
+    Meeting = apps.get_model("meeting", "Meeting")
+    Schedule = apps.get_model("meeting", "Schedule")
+    Session = apps.get_model("meeting", "Session")
+    SchedulingEvent = apps.get_model("meeting", "SchedulingEvent")
+    SchedTimeSessAssignment = apps.get_model("meeting", "SchedTimeSessAssignment")
+    TimeSlot = apps.get_model("meeting", "TimeSlot")
+
+    ntby = nametimes_by_year()
+    for year in ntby.keys():
```
Comment (jennifer-richards) [inline ietf/meeting/migrations/0005_iabminutes.py:24]: Minor nit: probably better to iterate over a dict with `for year in ntby` without the `.keys()`. (Does the same thing, but does not have to set up a dictionary view.)

**Review by jennifer-richards — COMMENTED** [review]
One (very minor) nit inline.

I wonder whether the migration is the right way to frame this update? You plan to run a management command to add the files. Would it make more sense to do this whole job with a management command? It's really more of a data injection than a migration, and splitting up the task between a migration and a management command invites things to get out of sync.

Edit to add: may want to add docevents to describe how the new items got into the database?

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
Couple minor things

File: ietf/meeting/management/commands/import_iab_minutes.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+from pathlib import PurePath
+
+from django.core.management.base import BaseCommand
+
+from ietf.doc.models import Document, DocAlias
+from ietf.meeting.models import Meeting, Schedule, Session, SchedulingEvent, SchedTimeSessAssignment, TimeSlot
+
+def nametimes_by_year():
+    with open(PurePath(__file__).parent.joinpath("data_for_import_iab_minutes"),"r") as file:
```
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/meeting/management/commands/import_iab_minutes.py:18]: Just trying to follow the documentation for PurePath vs Path - the literature is a mess around it. I'm happy to change as you suggest.

File: ietf/group/views.py [hunk 3]
```diff
@@ -833,11 +833,9 @@ def meetings(request, acronym=None, group_type=None):
 
     four_years_ago = timezone.now()-datetime.timedelta(days=4*365)
 
-    sessions = add_event_info_to_session_qs(
-        group.session_set.filter(
-            meeting__date__gt=four_years_ago,
-            type__in=['regular','plenary','other']
-        )
+    sessions = group.session_set.with_current_status().filter(
```
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/group/views.py:836]: So, I changed it here. It's not clear what the original intent of making acronym optional was, but the implementation clearly requires it.

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**